core/src/syscall/write.rs

```diff
@@ -16,7 +16,12 @@ impl Syscall for SyscallWrite {
         let a2 = Register::X12;
         let rt = &mut ctx.rt;
         let fd = arg1;
-        if fd == 1 || fd == 2 || fd == 3 || fd == 4 {
+        let mut maybe_hook = None;
+        if fd == 1 || fd == 2 || fd == 3 || fd == 4 || {
```

Comment:
nit: it's a bit unusual to have a block in the if clause. Also do we even need this if statement?